certbot/certbot/_internal/tests/cli_test.py

```diff
@@ -446,7 +445,7 @@ def test_deploy_hook_sets_renew_hook(self):
         assert_set_by_user_with_value(namespace, 'renew_hook', value)
 
     def test_renew_hook_conflict(self):
-        with mock.patch("certbot._internal.cli.sys.stderr"):
+        with mock.patch("sys.stderr"):
```

Comment:
I'm not really sure why `certbot._internal.cli.sys.stderr` is required, and not just `sys.stderr`